src/groupeddataframe/grouping.jl

```diff
@@ -94,11 +94,22 @@ groupby(d::AbstractDataFrame, cols;
         sort::Bool = false, skipmissing::Bool = false) =
     groupby(d, [cols], sort = sort, skipmissing = skipmissing)
 
-Base.start(gd::GroupedDataFrame) = 1
-Base.next(gd::GroupedDataFrame, state::Int) =
-    (view(gd.parent, gd.idx[gd.starts[state]:gd.ends[state]]),
-     state + 1)
-Base.done(gd::GroupedDataFrame, state::Int) = state > length(gd.starts)
+if VERSION < v"0.7.0-alpha"
```

Comment:
AFAICT the exact commit is 0.7.0-DEV.5126.

Reply:
Fixed. Thank you for being precise (the issue came out independently today https://discourse.julialang.org/t/maintaining-iterator-protocol-for-both-0-6-and-0-7/12084/4 😄).